lib/spree_multi_domain.rb

```diff
@@ -2,3 +2,4 @@
 require 'spree_multi_domain/engine'
 require 'sass/rails'
 require 'spree_extension'
+require 'webdrivers'
```

Comment:
Why do you add require 'webdrivers' here and remove it in the other commit?

Reply:
by merge with this [branch](https://github.com/spree-contrib/spree-multi-domain/tree/rails_6_spree_4) automatically added this `require 'webdrivers'` 

I squashed this commits.